src/deprecated.js

```diff
@@ -505,6 +506,10 @@ export var scene = {
     SkinInstance: SkinInstance
 };
 
+Morph.prototype.getTarget = function (index) {
+    return this._targets[index];
```

Comment:
```suggestion
return this.targets[index];
```

Reply:
So your code here is fine and it works. However, I've tweaked it so that the deprecated function implementation relies on the public API for `Morph` rather than directly access a private property. I've used this approach throughout `deprecated.js`.